### DISCUSSION TITLE
How can I replace the excerpt length through a fix text for each post in the Mellisa theme?

### DISCUSSION THREAD

**Discussion by ghost** [GetPublii/Publii#1671] [Help]
Good afternoon,
My question is: How can I replace the excerpt length through a fix text for each post in the Mellisa theme instead with the number of words?
Due to the different number of words in each post, there appear sometimes two lines of text, sometimes three. That makes the optics a little bit agitated. Sometimes these texts are also cut off in places that I don't like. I would like to add a short preview text for each post instead of the excerpt lenght standard text that is automatically taken from the post. 
Can anyone help me? I already tried with Chat GPT, but that didn't lead to a solution. 
Thank you very much. Have a lovely day.

**Comment by bobmitro** [comment]
Just read this article: [Excerpts with variable length](https://getpublii.com/dev/excerpts-with-variable-length/)

**Reply by ghost** [reply]
Oh, hi Bob. We already know each other. I found out how to ask a question here, as you can see.
In the Mellisa theme, I can define the number of words in the theme settins already. But I don't want any number of words to define the text, but a custom made text for each post.

**Reply by bobmitro** [reply]
> But I don't want any number of words to define the text, but a custom made text for each post.

So, when writing, use the "Read More" button. The content before this button will be your custom excerpt.

<img width="923" alt="screenshot 2024-09-06 at 15 33 00" src="https://github.com/user-attachments/assets/58752fa9-2e44-4ce3-914b-84a44f4fcc39">

**Comment by ghost** [comment]
Danke, Bob, das ist genau das, was ich gesucht habe.
